src/Modules/deRhamComplexes.jl

```diff
@@ -244,3 +244,57 @@ end
 function derivative(a::MPolyQuoRingElem, i::Int)
   return parent(a)(derivative(lift(a), i))
 end
+
+########################################################################
+# relative Kaehler differentials
+########################################################################
+
+function relative_kaehler_differentials(
+    R::MPolyAnyRing; 
+    base_variables::Vector{T}=elem_type(R)[]
+  ) where {T <: RingElem}
+  @assert all(x in gens(R) for x in base_variables) "given polynomials must be generators of the ring"
+  @assert _allunique(base_variables)
+  return _relative_kaehler_differentials(R, sort!([findfirst(==(y), gens(R)) for y in base_variables]))
+end
+
+function _relative_kaehler_differentials(
+    R::Union{MPolyRing, MPolyLocRing},
+    ind::Vector{Int}
+  )
+  n = ngens(R)
+  m = length(ind)
+  result = FreeMod(R, n-m)
+  ind_comp = [i for i in 1:n if !(i in ind)]
+  symb = symbols(R)[ind_comp]
+  result.S = [Symbol(:d, symb[i]) for i in 1:(n-m)]
+  #set_attribute!(result, :show, show_kaehler_differentials)
+  #set_attribute!(result, :is_kaehler_differential_module, (R, 1))
+  return result
+end
+
+function _relative_kaehler_differentials(
+    R::Union{MPolyQuoRing, MPolyQuoLocRing},
+    ind::Vector{Int}
+  )
+  n = ngens(R)
+  m = length(ind)
+  ind_comp = [i for i in 1:n if !(i in ind)]
+  ind_comp_inv = 
```

Comment:
```suggestion
```
? (I realize this code is WIP and so this may be in there intentionally as a placeholder; I hesitated if I should even mention this; in any case please feel free to completely ignore this)

Reply:
Yes, this is a WIP stub. Shouldn't be here.